modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -33,6 +33,8 @@ const RichText = ({
 
 	const editorRef = useRef();
 
+	editorConfig.contentsLangDirection = otherProps.dir;
```

Comment:
In react we should [never change a property value](https://reactjs.org/docs/components-and-props.html#props-are-read-only), because that can lead the application to an unexpected behaviour.

But you always can use it as base to create a new `const` to be passed to a child component:
```jsx
const newEditorConfig = {
	...editorConfig,
	contentsLangDirection: otherProps.dir,
}
```

Reply:
Thanks for the advice! Will work on it right away.